src/models/Settings.js

```diff
@@ -86,6 +86,7 @@ export default Model.extend({
     'features.multiOptionalFactorEnroll': ['boolean', true, false],
     'features.deviceFingerprinting': ['boolean', false, false],
     'features.hideSignOutLinkInMFA': ['boolean', false, false],
+    'features.skipIdpFactorVerificationBtn': ['boolean', false, false],
```

Comment:
Do we need to document this under our [features](https://github.com/okta/okta-signin-widget/#feature-flags)?

Reply:
@mohitganotra-okta can you please confirm?